client/state/comments/test/reducer.js

```diff
@@ -219,65 +162,33 @@ describe( 'reducer', () => {
 			const siteId = 1;
 			const requestId = createRequestId( siteId, postId, { after: new Date(), order: 'DESC', number: 10 } );
 
-			let action = {
+			const action = {
 				type: COMMENTS_REQUEST,
 				siteId,
 				postId,
 				requestId: requestId
 			};
 
-			let res = requests( undefined, action );
-			let specificRes = getPostCommentRequests( { comments: { requests: res } }, siteId, postId );
-
-			expect( specificRes.get( requestId ) ).to.be.eql( COMMENTS_REQUEST );
+			const response = requests( undefined, action );
+			expect( response[ '1-1' ][ requestId ] ).to.be.eql( COMMENTS_REQUEST );
 
 			action.type = COMMENTS_REQUEST_FAILURE;
-			res = requests( res, action );
-
-			specificRes = getPostCommentRequests( { comments: { requests: res } }, siteId, postId );
+			const failureResponse = requests( deepFreeze( response ), action );
 
-			expect( specificRes.get( requestId ) ).to.be.eql( COMMENTS_REQUEST_FAILURE );
+			expect( failureResponse[ '1-1' ][ requestId ] ).to.be.eql( COMMENTS_REQUEST_FAILURE );
 		} );
 	} ); // end of requests
 
 	describe( '#totalCommentsCount()', () => {
 		it( 'should update post comments count', () => {
-			const newState = totalCommentsCount( undefined, {
+			const response = totalCommentsCount( undefined, {
 				type: COMMENTS_COUNT_RECEIVE,
 				totalCommentsCount: 123,
 				siteId: 1,
 				postId: 1
 			} );
 
-			const specificRes = getPostTotalCommentsCount( { comments: { totalCommentsCount: newState } }, 1, 1 );
-
-			expect( specificRes ).to.eql( 123 );
+			expect( response[ '1-1' ] ).to.eql( 123 );
 		} );
 	} ); // end of totalCommentsCount
-
-	describe( '#comments()', () => {
```

Comment:
the plan is to immediately after this add a schema and move the reducer to a `keyedReducer` `withSchemaValidation`. For the moment we are opting out just by doing
```js
case SERIALIZE:
case DESERIALIZE:
    return {};
```